airflow/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -898,3 +898,71 @@ def execute(self, context: Context):
         ).fetch_cluster_info()
 
         return super().execute(context)
+
+
+class GKEStartKueueJobOperator(GKEStartJobOperator):
+    """
+    Executes a Kubernetes Job in Kueue in the specified Google Kubernetes Engine cluster.
```

Comment:
Is it "Kueue" or "Queue"? I see different words used in the operator name and operator parameters , for example.

Reply:
Ok, apparently it is intentional, disregard my previous comment.

Reply:
It is "Kueue". This word is a combination of two words Kubernetes and Queue :) 
More about it here: https://kueue.sigs.k8s.io/docs/overview/

Reply:
For parameter I use `queue_name` because it is a name of Queue(https://cloud.google.com/kubernetes-engine/docs/tutorials/kueue-intro#create_the_localqueue) which runs inside K8s use Kueue functionality.